jsx/Filter.js

```diff
@@ -88,6 +105,7 @@ class Filter extends Component {
           legend={this.props.title}
         >
           {this.renderFilterFields()}
+          <br/>
```

Comment:
is this line break required?

Reply:
umm, IIRC it was to standardize the placement of the clear filter button? Not necessary. I think both you and @driusan don't like it so we can find another solution at some point.